tools/opt/opt.cpp

```diff
@@ -239,6 +239,13 @@ Options (in lexicographical order):
   --print-all
                Print SPIR-V assembly to standard error output before each pass
                and after the last pass.
+  --time-report
```

Comment:
Please move this down to preserve lexicographical order of the flag names.

Reply:
Done